fabtests/pytest/conftest.py

```diff
@@ -101,7 +101,7 @@ def populate_command(self, base_command, host_type, timeout=None):
             command = self._populate_ubertest_command(command, host_type)
         elif command.find("fi_multinode") != -1:
             command = self._populate_multinode_command(command, host_type)
-        else:
+        elif command.find("fi_") != -1:
```

Comment:
Not sure this is necessary. Does it cause any harm?

Reply:
There are some tests that does not start with `fi_`

Reply:
I see. Thanks for pointing that out.

It's not a bug. I wanted to reuse `populate_command` to execute `check_hmem` which does not support the same fabtest parameters.

I guess that's fine. I don't have to use this. Will update the change.